pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [project]
 name = "dbt-semantic-interfaces"
-version = "0.6.3"
+version = "0.6.4"
```

Comment:
Can you move this to a separate PR?

Reply:
Shoot, sorry got Will's approval and merged before I got this 😅